config/routes.rb

```diff
@@ -6,8 +6,7 @@
     resources :contacts
   end
 
-  resources :characteristics, only: [:show]
-
+  resources :characteristics
```

Comment:
When I do that, I can no longer access characteristics_path, which I used to implement the delete method, I will keep it for now.